SparkPost/2web2ui
# SA-2042: Report Builder update for timezones

### What Changed

-Datepicker updates from timezone
- Manual entry form properly parses timezone
- Datepicker displayed timezone changed

### How To Test

- Go to report builder
- Change timezone
- Verify that dates in datepicker change
- Verify that to extends to specified timezone instead of local timezone
- Verify that chart pulls date/time for timezone (in request
- Verify chart labelling is correct

### To Do
